pkg/scm/gitlab/client_actioner.go

```diff
@@ -153,6 +155,81 @@ func (c *Client) ApplyStep(ctx context.Context, evalContext scm.EvalContext, upd
 
 		return err
 
+	case "assign_reviewers":
+		source, err := step.RequiredStringEnum("source", "codeowners")
+		if err != nil {
+			return err
+		}
+
+		desiredLimit, err := step.RequiredInt("limit")
+		if err != nil {
+			return err
+		}
+
+		mode, err := step.OptionalStringEnum("mode", "linear", "linear", "random")
```

Comment:
Whats the use-case for `mode=linear`? Won't it always select the (arbitrary) first X users from the legibility list?

Reply:
it will, might not be too useful unless the is a distinct hierarchy in the order of the owners in the entry, where the top X users that own it will get pinged. but those use cases might not be as common

Reply:
removed until necessary